### PR TITLE
added [continuousphp] integration

### PR DESCRIPTION
**Pull request by ppaulis**
https://continuousphp.com

### CODE REVIEW

**Comment by Pierozi** [comment]
Thanks pascal, the Doctrine maintainers will love that.

File: server.js [hunk 1]
```diff
@@ -393,6 +393,66 @@ cache(function(data, match, sendBadge, request) {
...
+  };
+
+  if (branch != null) {
+    options.uri += '?branch=' + branch;
+  }
+
+  var badgeData = getBadgeData('build', data);
+  request(options, function(err, res) {
+    if (err != null) {
+      console.error('continuousphp error: ' + err.stack);
```
Comment (Daniel15) [inline server.js]: Were these `console.error` calls only for debugging, or do you want to keep them at runtime too?

Comment (ppaulis) [reply server.js]: Hello @Daniel15, thanks for reviewing our PR! If that's ok I'd like to keep the error handling. If there is another design implementation for the error handling that you would prefer, don't hesitate to get back to me :-)

**Comment by Pierozi** [comment]
Hello, any update? I will love to have this badge!

**Comment by paulmelnikow** [comment]
Thanks!

**Comment by ppaulis** [comment]
@paulmelnikow @Daniel15 many thanks to you guys!

**Comment by paulmelnikow** [comment]
And thanks to @espadrine for deploying!

**Pull request merged**